run_functional_tests.lua

```diff
@@ -146,11 +148,11 @@ local function adjustFile( fileOut, fileIn, pattern, mayBeAbsent, verbose )
             if verbose then
                 print('Modifying line: '..line )
             end
-            line = string_sub(line, dest, source)
+            line = string_gsub(line, dest, source)
             -- line = line:sub(1,idxStart-1)..source..line:sub(idxEnd+1)
             -- string.gsub( line, dest, source )
             if verbose then
-                print('Result: '..line )
+                print('Result        : '..line )
```

Comment:
Ah, I got it. It brings the line's prefix up to the same length as `'Modifying line: '` further up, for aligment / easier comparison.